src/config.ts

```diff
@@ -10,10 +10,13 @@ export interface Config {
   locale?: string
   /** Export all blocks of the DB/page or just the ones in the current view. Default: "all" */
   collectionViewExportType?: "currentView" | "all"
+  /** Poll export task finished interval in ms */
+  pollInterval: number
```

Comment:
Let's actually leave this as optional because otherwise, one always has to provide it. 
(Also it causes a TS error in the testing pipeline...)

Reply:
Sorry about [that](https://github.com/yannbolliger/notion-exporter/actions/runs/9224775012/job/25426053845?pr=9) 😞 -> I'll review (carefully) and fix it